app/templates/rules.html

```diff
@@ -84,7 +84,12 @@ <h3 class="panel-title">
     <div class="left" style="float: left">
 
       <h5 title="Mapping">
-        Mapping: <b ng-bind-html="highlightSearch(rule.mapping, 'mapping')"></b>
+        Mapping: <span ng-show="rule.mapping">
```

Comment:
Yeah `Mapping: ` looks odd. I'd rather it disappeared or said explicitly that there isn't a mapping. Also, I personally always avoid the word "None" since it looks like either a name (because it's capitalized) or the python word of null. I often put `<i>none</i>`.